xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-page/src/test/java/org/xwiki/test/page/HTML50ComponentList.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.test.page;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.xwiki.rendering.internal.macro.html.renderers.html5.HTMLMacroHTML5Renderer;
+import org.xwiki.rendering.internal.renderer.html5.HTML5BlockRenderer;
+import org.xwiki.rendering.internal.renderer.html5.HTML5Renderer;
+import org.xwiki.rendering.internal.renderer.html5.HTML5RendererFactory;
+import org.xwiki.rendering.internal.renderer.xhtml.image.DefaultXHTMLImageRenderer;
+import org.xwiki.rendering.internal.renderer.xhtml.image.DefaultXHTMLImageTypeRenderer;
+import org.xwiki.rendering.internal.renderer.xhtml.link.DefaultXHTMLLinkRenderer;
+import org.xwiki.rendering.internal.renderer.xhtml.link.DefaultXHTMLLinkTypeRenderer;
+import org.xwiki.test.annotation.ComponentList;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Pack of default Component implementations that are needed for Parsing and Rendering in HTML 5.0.
+ *
+ * @version $Id$
+ * @since 13.2RC1
```

Comment:
@vmassol I guess we'll apply this change to 12.10.x too?

Reply:
You mean this whole PR? We could.

Reply:
ok, in this case I should add a `@since 12.10.6`

Reply:
done